src/common/transformations/src/transformations/common_optimizations/rms_fusion.cpp

```diff
@@ -67,8 +67,11 @@ RMSFusion::RMSFusion() {
     auto gamma = wrap_type<ov::op::v0::Constant>(type_matches(element::f32));
     auto mul2 = wrap_type<ov::op::v1::Multiply>({gamma, mul1});
 
-    // compress RMS result
-    auto comp = wrap_type<ov::op::v0::Convert>({mul2});
+    std::shared_ptr<ov::Node> comp = mul2;
+    if (force_tail_convert) {
+        // compress RMS result
+        comp = wrap_type<ov::op::v0::Convert>({mul2});
+    }
```

Comment:
Why `force_tail_convert` is introduced? is it because `Convert` node is not mandatory, it's not exist on some model?

if so, why not use `mul2` as root and detect this optional `Convert` inside callback?
since RMS support any output_type, `convert` should be optional instead of mandatory

Reply:
I've tried checking `mul2` in the [commit](https://github.com/openvinotoolkit/openvino/pull/26147/commits/08bbaac0ae4efb5216549a817f6bba03c7de1384) but failed in GPU testcase. It seems that GPU needs this `convert` node mandatory. In original model there is no `convert` node so it should be safe if it's removed from the pattern for CPU.